v4/platformUI/src/main/java/exchange/dydx/platformui/components/container/PlatformInfoContainer.kt

```diff
@@ -173,6 +249,10 @@ class PlatformInfo @Inject constructor(
                     displayNextToast()
                 } catch (e: CancellationException) {
                     _toasts.value = null
+                    appScope.launch {
+                        delay(500)
```

Comment:
Delay is needed otherwise, the next toast won't show probably due to the animation delay.  Let me know if you have a better solution here. @prashanDYDX

Reply:
let's merge for now, i'll look into this